.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  package_name: <YOUR PACKAGE NAME HERE>
+
+jobs:
+  version_check:
+    runs-on: ubuntu-latest
+    outputs:
+      v_package: ${{ steps.version.outputs.dbt_package_version}}
+      v_old: ${{ steps.previoustag.outputs.tag}}
+      new_version: ${{ steps.new_version_flag.outputs.new_version}}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+          fetch-depth: 0
+
+    - name: 'Get Previous tag'
+      id: previoustag
+      uses: "WyriHaximus/github-action-get-previous-tag@v1"
+      with:
+        fallback: 0.0.1 # Optional fallback tag to use when no tag can be found
+
+    - name: Get tag and package versions
+      id: version
+      run: |
+        echo "dbt_package_version=$(grep -oP "(?<=version: ')(.*)(?=')" dbt_project.yml)" >> $GITHUB_OUTPUT
```

Comment:
So this is like regex matching the `version` attribute from the `dbt_project.yml` file?

Reply:
Indeed